src/Sylius/Bundle/CoreBundle/Resources/config/services/mailer.xml

```diff
@@ -40,5 +44,23 @@
         <argument type="service" id="sylius.email_sender" />
     </service>
     <service id="Sylius\Bundle\CoreBundle\Mailer\ShipmentEmailManagerInterface" alias="sylius.mailer.shipment_email_manager" />
+
+    <service id="Sylius\Bundle\CoreBundle\Mailer\ResetPasswordEmailManagerInterface"
+             class="Sylius\Bundle\CoreBundle\Mailer\ResetPasswordEmailManager"
+    >
+        <argument type="service" id="sylius.email_sender" />
+    </service>
```

Comment:
This is not yet written down anywhere as an ADR or at least a convention, but, as with the other email managers in this file, define them with names according to the template `sylius.mailer.x_email_manager` and add an alias for FQCN